pkg/database/vercode.go

```diff
@@ -110,6 +110,16 @@ func (db *Database) FindVerificationCode(code string) (*VerificationCode, error)
 	return &vc, nil
 }
 
+// FindVerificationCodeByIDs find a list of verification codes by UUID.
+func (db *Database) FindVerificationCodeByIDs(uuids []string) ([]VerificationCode, error) {
```

Comment:
This feels expensive and is also untested. Would we every query by more than one?

Reply:
it shouldn't be expensive - it is an index query and should be no more expensive in the single ID case, but it should be cheaper than repeated queries. We will want to cap the length of the request batch size.

I'm following up in the issue to ensure that the user requirement is batch oriented.